docs/source/migration/3.0.mdx

```diff
@@ -623,6 +623,15 @@ val apolloClient = ApolloClient.Builder()
     .build()
 ```
 
+The [idlingResource function](https://github.com/apollographql/apollo-kotlin/blob/v3.0.0-rc03/apollo-idling-resource/src/main/java/com/apollographql/apollo3/android/ApolloIdlingResource.kt#L60-L63) is hosted under the new artifact `apollo-idling-resource`.
+Add the new dependency like so:
+
+```kotlin
+dependencies {
+  testImplementation("com.apollographql.apollo3:apollo-idling-resource:$version")
```

Comment:
I think this used to be at `com.apollographql.apollo:apollo-idling-resource` so it didn't change much? Or were you using something else?

Reply:
You are 100% correct. While doing the migration I had accidentally deleted the old implementation so I thought that this one was a similar case like how `apollo-http-cache` was added. Sorry!